component/app.jsonnet

```
@@ -3,7 +3,15 @@ local inv = kap.inventory();
 local params = inv.parameters.namespaces;
 local argocd = import 'lib/argocd.libjsonnet';
 
-local app = argocd.App('namespaces', params.namespace);
+local app = argocd.App('namespaces', params.namespace) {
```

App name must be the instance name for multi-team support to work correctly.

```suggestion
local instance = inv.parameters._instance;

local app = argocd.App(instance, params.namespace) {
```

You also need to use the instance name on line 21 to avoid issues when we have multiple instances assigned to the same team for some reason.

```
 ['%s/%s' % [appPath, instance]]: app,
```